api/tests/conftest.py

```diff
@@ -319,3 +322,17 @@ class implementation.
 
         # Force the deletes to the DB
         db_session.commit()
+
+    @pytest.fixture(scope="class")
+    def truncate_staging_tables(self, db_session):
+        for table in staging_metadata.tables.values():
+            db_session.execute(delete(table))
+
+        db_session.commit()
+
+    @pytest.fixture(scope="class")
+    def truncate_foreign_tables(self, db_session):
+        for table in foreign_metadata.tables.values():
+            db_session.execute(delete(table))
```

Comment:
Not sure if it's worth the trouble, but doing TRUNCATE TABLE is often way more efficient (skips writing the DELETEs to the log) for large tables. From my poking around it seems like SQLAlchemy doesn't support it via it's API (which makes sense as not every version of every DB engine supports it) but we could run it as a raw SQL if we wanted to go that route.

Reply:
Only a minor change to do - it works fine in this case.